content/development/getting-started.html

```diff
@@ -17,7 +20,32 @@ <h3>Getting Involved</h3>
 and preferences are. Usually, by running Haiku for a while you will easily find
 out things that don't work as expected or that can be improved.
 
-<!--more-->
+<h2><a name="development_tools" title="development_tools"></a>What do I need for coding?</h2>
+<ul>
+	<li>
+		Currently, Haiku is the most optimal and frictionless development environment.
+		Linux, BSD, and OS X also work perfectly fine.
+                BeOS, Zeta, Cygwin, and Solaris are no longer actively maintained as development environments and little to no support is offered for those platforms. 
+	</li>
+	<li>
+		Subscribe to these mailing lists:
+		<ul>
+			<li>The <a href="https://www.freelists.org/list/haiku-development">development mailing list</a></li>
+			<li>The <a href="https://www.freelists.org/list/haiku">general Haiku mailing list</a></li>
+			<li><a href="https://www.freelists.org/list/haiku-commits">Commit logs</a></li>
+		</ul>
+	</li>
+	<li>Learn which <a href="/guides/building/pre-reqs">development tools</a> are needed on your platform.
+</li>
+	<li>Read how to <a href="/guides/building">get and build our source code</a>.</li>
```

Comment:
a lot of this is specific to modifying Haiku itself, but the haiku-specific section starts right below. Seems a bit confusing to me